input/resources/valueset-rsg-source-document-type.xml

```diff
@@ -12,11 +12,11 @@
         <system value="urn:ietf:rfc:3986"/>
         <value value="urn:oid:2.16.840.1.113883.2.3.4.2.4.32"/>
     </identifier>
-    <name value="CommonAURecordedSexOrGenderSourceDocumentType" />
-    <title value="Common AU Recorded Sex or Gender (RSG) Source Document Type" />
+    <name value="AURecordedSexOrGenderSourceDocumentType" />
+    <title value="AU Recorded Sex or Gender (RSG) Source Document Type" />
     <status value="draft" />
     <experimental value="false"/>
-    <description value="The Common AU Recorded Sex or Gender (RSG) Source Document Type value set includes codes that can be used to represent the type of document where this sex or gender value is recorded." />
+    <description value="The AU Recorded Sex or Gender (RSG) Source Document Type value set includes codes that can be used to represent the type of document where this sex or gender value is recorded." />
```

Comment:
This change is not related to FHIR-46107, please revert.

Reply:
Reverted change

Reply:
Change reverted